kitsune/wiki/views.py

```diff
@@ -135,16 +135,11 @@ def document(request, document_slug, template=None, document=None):
 
     # Render the static pages if the slug matches the experiment
     if (flag_is_active(request, 'ux_experiment_1') and
+        switch_is_active('ux_experiment_1') and
```

Comment:
NIT: I would reverse these. The switch is for the whole thing, and the flag will do the segmentation. No sense checking the flag if the switch is off.